docs/src/languages/deno.md

```diff
@@ -10,6 +10,15 @@ To use the Deno Language Server with TypeScript and TSX files, you will likely w
 
 ```json
 {
+  "lsp": {
+    "deno": {
+      "settings": {
+        "deno": {
+          "enable": true
```

Comment:
Is this something that we should just enable out of the box rather than require everyone to add it to their settings?

Also, could would you mind referring me to the Deno docs that talk about what this setting does?

Reply:
I don't think that there is something to do here, deno works in all the other editors the same way, I think its made this way so the user get to choose if this is a deno or node project.

In vscode its more convenient because the deno extension, will automatically notify the user (based on some heuristics like if deno.json is present) and ask him if he wants to enable deno, and it changes the setting automatically

But In practice its not really a big deal, it works well as is

the deno lsp is documented here https://docs.deno.com/runtime/reference/lsp/

Reply:
Here is the description for that setting https://github.com/denoland/vscode_deno/blob/main/package.json#L186

You can find there the rest of possible properties

Reply:
Ah, okay, thank you for the explanation!

I'll go ahead and get this merged 🙂